hack/lib/mesh_resources/smcp.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: basic
   namespace: istio-system
 spec:
+  techPreview:
+    meshConfig:
+      defaultConfig:
+        terminationDrainDuration: 35s
```

Comment:
Just FYI, we also deploy this webhook for mesh tests which sets the timeout to 20 seconds: https://github.com/openshift-knative/serverless-operator/blob/main/serving/metadata-webhook/pkg/defaults/ksvc_defaults.go#L19
It probably overrides what you define here.

Reply:
Yeah I've seen that, but it does not seem to work. Envoy still terminates after seconds. That flag seems to work. We'll need to dig into that as well.

Reply:
From https://gist.github.com/ReToCode/e0b2a8d0d91809bf489caa6e26d287ca#file-envoy-proxy-terminates-too-early-txt
```
2023-07-13T09:10:28.706951023Z istio-proxy 2023-07-13T09:10:28.706875Z    warn    Envoy proxy is NOT ready: server is not live, current state is: DRAINING
2023-07-13T09:10:30.707229025Z istio-proxy 2023-07-13T09:10:30.707137Z    warn    Envoy proxy is NOT ready: server is not live, current state is: DRAINING
2023-07-13T09:10:32.706613579Z istio-proxy 2023-07-13T09:10:32.706571Z    warn    Envoy proxy is NOT ready: server is not live, current state is: DRAINING
2023-07-13T09:10:32.768848101Z istio-proxy 2023-07-13T09:10:32.768775Z    warn    Aborted proxy instance
2023-07-13T09:10:32.768848101Z istio-proxy 2023-07-13T09:10:32.768811Z    warn    Aborting proxy
```

Reply:
Ah. That was the DestroyPodInflight failure. Thanks for the info.

Reply:
👍 more details here: https://issues.redhat.com/browse/SRVKS-1099